apps/charterafrica/src/pages/[slug]/[slug].spec.js

```diff
@@ -0,0 +1,7 @@
+const { test, expect } = require("@playwright/test");
```

Comment:
Very good thinking... as you can see my WIP in #333 , we can take this further and actually delete all pages, leaving just `[...slug].page.js` and `index.page.js`.